dockstore-integration-testing/src/test/java/io/dockstore/common/CommonTestUtilities.java

```diff
@@ -175,16 +192,47 @@ private static String getDockstoreToken(TestingPostgres testingPostgres, String
         return "Bearer " + (testingPostgres
                 .runSelectStatement("select content from token where tokensource='dockstore' and username= '" + username + "';", String.class));
     }
+    /**
+     *  Deletes BitBucket Tokens from Database
+     *
+     *  @param testingPostgres reference to the testing instance of Postgres
+     * @throws Exception
+     */
+    public static void deleteBitBucketToken(TestingPostgres testingPostgres)  {
+        if (testingPostgres != null) {
+            LOG.info("Deleting BitBucket Token from Database");
+            testingPostgres.runUpdateStatement("delete from token where tokensource = 'bitbucket.org'");
+        } else {
+            LOG.info("testingPostgres is null");
```

Comment:
A small number of tests have `testingPostgres = null`, so this check ensures that `testingPostgres.runUpdateStatement("delete from token where tokensource = 'bitbucket.org'");` is only run on databases that are not `null`.

I considered providing the option to have functions like `cleanStatePrivate1(...)` not require `testingPostgres`, although I fear that this option may lead to the BitBucket tokens being left in the database unintentionally.